### PR TITLE
Guidelines: Expand on "deprecations are not for cleanup"

### PR DESCRIPTION
**Pull request by tordans**
This topic came up a couple of times recently and I think it would be good to have a section on this in the docs.

My goal is, to have something to link to when a deprecation rule is added so that PR contributors know that the deprecation rules might not be the tool they are looking for.

This addition likely needs some polishing and I hope for input…

### CODE REVIEW

File: GUIDELINES.md [hunk 1]
```diff
@@ -37,11 +37,21 @@ No preset or field is isolated; they are always presented alongside others in va
...
 - There is a significant drop in usage compared to previous numbers, with a negative trend ([visible in the graph](https://taghistory.raifer.tech/)).
 - Usage of the deprecated tag remains stagnant for a longer period (about a year).
 
+**Deprecations are not for cleanup:**
+
+Deprecation rules work such that the user sees a message with suggestions and can act only when editing the given element. This makes them well-suited for gradual, human-reviewed updates of taggings like crossings. However, they are not suitable for cleaning up incorrect tagging from the database, especially for low-volume changes.
+
+There are, however, alternatives to consider: 
+- Your cleanup task might be eligible for an automated (bot) edit. [Please learn more on the wiki…](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct)
+- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick.
```
Comment (matkoniecz) [inline GUIDELINES.md:52]: maybe it should be made clear that you should consult community? and mass changing tag in JOSM/level0 is still an automated edit?

File: GUIDELINES.md [hunk 2]
```diff
@@ -37,11 +37,21 @@ No preset or field is isolated; they are always presented alongside others in va
...
+- A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/). For small changes, a [Cooperative Challenge](https://learn.maproulette.org/en-us/documentation/creating-cooperative-challenges/) will likely be a good fit, allowing mappers to change the tagging directly from MapRoulette.
```
Comment (matkoniecz) [inline GUIDELINES.md:53]: note that it is not really a good way for anything that requires actual review but has an obvious "fix"

too many people edit blindly or use MR as an automated edit, run via blindly made human edits

File: GUIDELINES.md (hunk 1 above)
Comment (matkoniecz) [reply GUIDELINES.md:52]:
```suggestion
- If your automated edit is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick - just remember to ask other mappers first, as required for such edits.
```

Comment (tordans) [inline GUIDELINES.md:52]:
```suggestion
- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. This is still an automated edit, though, so the [Guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please talk to other mappers first.
```

File: GUIDELINES.md (hunk 2 above)
Comment (tordans) [inline GUIDELINES.md:53]:
```suggestion
- A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/).
```

**Review by tordans — COMMENTED** [review]
(Removed)

File: GUIDELINES.md (hunk 2 above)
Comment (tordans) [reply GUIDELINES.md:53]: Agreed, the feature is not there yet and it's better to be cautious here.

File: GUIDELINES.md (hunk 1 above)
Comment (tordans) [reply GUIDELINES.md:52]: Thanks, I committed a modified version of this.

File: GUIDELINES.md [hunk 3]
```diff
@@ -37,11 +37,21 @@ No preset or field is isolated; they are always presented alongside others in va
...
+
 - The deprecation is documented in the wiki and is either official (resulting from a proposal process) or long-standing (about a year).
 - There is a significant drop in usage compared to previous numbers, with a negative trend ([visible in the graph](https://taghistory.raifer.tech/)).
 - Usage of the deprecated tag remains stagnant for a longer period (about a year).
 
+**Deprecations are not for cleanup:**
+
+Deprecation rules work such that the user sees a message with suggestions and can act only when editing the given element. This makes them well-suited for gradual, human-reviewed updates of taggings like crossings. However, they are not suitable for cleaning up incorrect tagging from the database, especially for low-volume changes.
+
+There are, however, alternatives to consider:
```
Comment (matkoniecz) [inline GUIDELINES.md:50]:
```suggestion
There are, however, alternatives to consider:
- You can simply resurvey objects in your area, sometimes mapping based on aerial imagery and other resources available for OpenStreetMap mapping
- You can contact mappers (via changeset comments) and local communities (by creating notes, [writing on local community forums](https://wiki.openstreetmap.org/wiki/Contact_channels) and suggest looking at their local areas
```

File: GUIDELINES.md [hunk 4]
```diff
@@ -37,11 +37,21 @@ No preset or field is isolated; they are always presented alongside others in va
...
 - Usage of the deprecated tag remains stagnant for a longer period (about a year).
 
+**Deprecations are not for cleanup:**
+
+Deprecation rules work such that the user sees a message with suggestions and can act only when editing the given element. This makes them well-suited for gradual, human-reviewed updates of taggings like crossings. However, they are not suitable for cleaning up incorrect tagging from the database, especially for low-volume changes.
+
+There are, however, alternatives to consider: 
+- Your cleanup task might be eligible for an automated (bot) edit. [Please learn more on the wiki…](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct)
+- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. This is still an automated edit, though, so the [Guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please talk to other mappers first.
+- A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/).
```
Comment (matkoniecz) [inline GUIDELINES.md:53]:
```suggestion
- A way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/), but someone creating such challenge should provide good instruction, ensure that mappers are not editing blindly and repair any damage caused by it. Consider making challenge private and sharing it in controlled fashion.
```

I would consider dropping it altogether. Or explicitly warn that some people will go through random MR challenges blindly editing without actual verification or thinking.

File: GUIDELINES.md [hunk 5]
```diff
@@ -37,11 +37,21 @@ No preset or field is isolated; they are always presented alongside others in va
...
 - There is a significant drop in usage compared to previous numbers, with a negative trend ([visible in the graph](https://taghistory.raifer.tech/)).
 - Usage of the deprecated tag remains stagnant for a longer period (about a year).
 
+**Deprecations are not for cleanup:**
+
+Deprecation rules work such that the user sees a message with suggestions and can act only when editing the given element. This makes them well-suited for gradual, human-reviewed updates of taggings like crossings. However, they are not suitable for cleaning up incorrect tagging from the database, especially for low-volume changes.
+
+There are, however, alternatives to consider: 
+- Your cleanup task might be eligible for an automated (bot) edit. [Please learn more on the wiki…](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct)
+- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. This is still an automated edit, though, so the [Guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please talk to other mappers first.
```
Comment (matkoniecz) [inline GUIDELINES.md:52]:
```suggestion
- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. Mass-replace without checking every single object is still an automated edit, though, so the [Guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please talk to other mappers first.
```

File: GUIDELINES.md (hunk 3 above)
Comment (matkoniecz) [reply GUIDELINES.md:50]: maybe also note that checking area distribution and sample of edits may be a good idea? sometimes it is a single edit or single mapper

sometimes it is even revertable as bogus import or vandalism

File: GUIDELINES.md (hunk 5 above)
Comment (tordans) [reply GUIDELINES.md:52]: Good addition. ChatGPT version is…

```suggestion
- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. However, mass-replacing without checking each object is still considered an automated edit, so the [guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please consult other mappers first.
```

File: GUIDELINES.md [hunk 6]
```diff
@@ -37,11 +37,21 @@ No preset or field is isolated; they are always presented alongside others in va
...
 - The deprecation is documented in the wiki and is either official (resulting from a proposal process) or long-standing (about a year).
 - There is a significant drop in usage compared to previous numbers, with a negative trend ([visible in the graph](https://taghistory.raifer.tech/)).
 - Usage of the deprecated tag remains stagnant for a longer period (about a year).
 
+**Deprecations are not for cleanup:**
+
+Deprecation rules work such that the user sees a message with suggestions and can act only when editing the given element. This makes them well-suited for gradual, human-reviewed updates of taggings like crossings. However, they are not suitable for cleaning up incorrect tagging from the database, especially for low-volume changes.
+
+There are, however, alternatives to consider: 
+- Your cleanup task might be eligible for an automated (bot) edit. [Please learn more on the wiki…](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct)
```
Comment (matkoniecz) [inline GUIDELINES.md:51]:
```suggestion
- Your cleanup task might be eligible for an automated (bot) edit. [Please learn more on the wiki…](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct) If mass-replace, using script or JOSM/level0 is beyond your abilities you can still suggest such edit at [OSM forums](https://community.openstreetmap.org/)
```

File: GUIDELINES.md (hunk 4 above)
Comment (tordans) [reply GUIDELINES.md:53]: No, I think it is the responsibility of MR to guide users to using the tool right. The role of this section is only to give starting points for further steps elsewhere…

File: GUIDELINES.md (hunk 3 above)
Comment (tordans) [reply GUIDELINES.md:50]: I agree those are good things to do. But the cases when I link to this guideline to close issues or ask for clarification seem to not fit into this category of local or single user edits.

Maybe we should add a wiki page or a section on the organized editing guidlines to go into more details? This question of "how to cleanup values" should be common outside of this repo, right?

File: GUIDELINES.md (hunk 4 above)
Comment (tordans) [inline GUIDELINES.md:53]:
```suggestion
- A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/).

Should those options be not for you, you can always suggest such changes in the [OSM community forum](https://community.openstreetmap.org/).
```

File: GUIDELINES.md (hunk 6 above)
Comment (tordans) [reply GUIDELINES.md:51]: Good idea, I added it to the end of the list separately.

File: GUIDELINES.md (hunk 3 above)
Comment (matkoniecz) [reply GUIDELINES.md:50]: > But the cases when I link to this guideline to close issues or ask for clarification seem to not fit into this category of local or single user edits.

ok, I was unaware of that, then it makes no sense

> Maybe we should add a wiki page or a section on the organized editing guidlines to go into more details? This question of "how to cleanup values" should be common outside of this repo, right?

+1 though I have no ready link - but if that case does not exist at problem at this repo we may simply not mention it

**Review by tyrasd — APPROVED** [review]

**Pull request merged**